manifests/0000_90_cluster-kube-apiserver-operator_03_servicemonitor.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: cluster-kube-apiserver-operator
+  namespace: openshift-cluster-kube-apiserver-operator
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 30s
+    metricRelabelings:
+    - action: drop
+      regex: etcd_(debugging|disk|request|server).*
+      sourceLabels:
+      - __name__
+    port: https
+    scheme: https
+    tlsConfig:
+      # TODO where do you mount the service-ca.crt?
```

Comment:
@openshift/sig-monitoring to monitor our operators securely, you need to have access to the service-ca bundle from @openshift/sig-auth

Reply:
@deads2k just rechecked, we are already mounting the service-ca bundle into prometheus, example config is here: https://github.com/openshift/cluster-monitoring-operator/blob/master/assets/kube-state-metrics/service-monitor.yaml#L16-L18